soc/v2/rockchip/Kconfig.soc

```diff
@@ -0,0 +1,13 @@
+#
+# Copyright 2021 Huawei France Technologies SASU
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+config SOC_FAMILY_ROCKCHIP
```

Comment:
this file is not sourced because this folder doesn't contain a soc.yml (and the Kconfig.soc in the folders containing a soc.yml are also not sourcing it).

Reply:
Fixed